packages/phone-number-privacy/signer/src/pnp/services/request-service.ts

```diff
@@ -69,6 +74,18 @@ export class DefaultPnpRequestService implements PnpRequestService {
       return undefined
     }
   }
+
+  public async removeOldRequest(daysToKeep: number, ctx: Context): Promise<Number> {
+    if (daysToKeep < 0) {
+      ctx.logger.error('RemoveOldRequest - DaysToKeep should be bigger than or equal to zero')
+    }
+    const since: Date = new Date(Date.now() - daysToKeep * 24 * 60 * 60 * 1000)
+    return traceAsyncFunction('DefaultPnpRequestService - removeOldRequest', () =>
+      this.db.transaction(
+        async (trx) => await deleteRequestsOlderThan(this.db, since, ctx.logger, trx)
```

Comment:
I can't think of a reason why we'd need a trx here. Maybe I'm missing something?

Reply:
you can call it without it, I left that as a simple way to make a cascade delete if we need it to remove another rows from another table